src/backend/parser/gram.y

```diff
@@ -18612,9 +18612,34 @@ SplitColQualList(List *qualList,
 		}
 		if (IsA(n, CollateClause))
 		{
+			const char *dump_restore = GetConfigOption("babelfishpg_tsql.dump_restore", true, false);
 			CollateClause *c = (CollateClause *) n;
 
-			if (*collClause)
+			/*
+			 * XXX: This is temporary fix.
+			 * During upgrade from 14.6 to 15.1, we can ignore extra COLLATE "default" clause.
```

Comment:
We should not mention specific engine versions since we don't know in which release this will get fixed.